documentation/src/main/asciidoc/topics/proc_setting_persistent_location.adoc

```diff
@@ -40,8 +37,33 @@ public GlobalStateConfigurationBuilder persistentLocation(String path, String re
 }
 ----
 
-{brandname} logs the following message if you attempt to use a directory
-outside the global persistent location with file-based cache stores:
+.File-Based Cache Stores and Global Persistent Location
+
+When using file-based cache stores, you can optionally specify filesystem directories for storage. These paths are relative to the global persistent location.
+
+For example, you configure your global persistent location as follows:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<global-state>
+   <persistent-location path="/tmp/example" relative-to="my.data"/>
+</global-state>
+----
+
+You then configure a Single File cache store that uses a path named
+`myDataStore` as follows:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<global-state>
+   <persistent-location path="myDataStore" relative-to="my.data"/>
+</global-state>
```

Comment:
This should be the `file-store` xml I think

Reply:
yes it should. nice catch.